input/tests/measure/UrinarySymptomScoreChangeAfterBenignProstaticHyperplasiaFHIR/623a7f40-c265-4e42-a5c5-6f54e20b19df/CMS771FHIR-v1.3.000-DENEXFail-UrinaryRetentionDxUnconfirmed.json

```diff
@@ -0,0 +1,363 @@
+    "resource": {
+      "resourceType": "Patient",
+      "id": "623a7f40-c265-4e42-a5c5-6f54e20b19df",
+      "meta": {
+        "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-patient" ]
+      },
+      "extension": [ {
+        "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+        "extension": [ {
+          "url": "ombCategory",
+          "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2054-5",
+            "display": "Black or African American"
+          }
+        }, {
+          "url": "text",
+          "valueString": "Black or African American"
+        } ]
+      }, {
+        "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
+        "extension": [ {
+          "url": "ombCategory",
+          "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2135-2",
+            "display": "Hispanic or Latino",
+            "userSelected": true
+          }
+        }, {
+          "url": "text",
+          "valueString": "Hispanic or Latino"
+        } ]
+      } ],
+      "identifier": [ {
+        "type": {
+          "coding": [ {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code": "MR"
+          } ]
+        },
+        "system": "https://GoodHealthCare.com",
+        "value": "960082319426"
+      } ],
+      "name": [ {
+        "family": "DENEXFail",
+        "given": [ "UrinaryRetentionDxUnconfirmed" ]
+      } ],
+      "gender": "male",
+      "birthDate": "1969-01-01"
+    },
+    "request": {
+      "method": "POST",
+      "url": "Patient/623a7f40-c265-4e42-a5c5-6f54e20b19df"
+    }
+  }, {
+    "fullUrl": "https://madie.cms.gov/Encounter/55509231",
+    "resource": {
+      "resourceType": "Encounter",
+      "id": "55509231",
+      "meta": {
+        "profile": [ "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-encounter" ]
+      },
+      "status": "finished",
+      "class": {
+        "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+        "code": "AMB",
+        "display": "ambulatory"
+      },
+      "type": [ {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "version": "http://snomed.info/sct/731000124108/version/202303",
+          "code": "185464004",
+          "display": "Out of hours visit - not night visit (procedure)",
+          "userSelected": false
+        } ]
+      } ],
+      "subject": {
+        "reference": "Patient/623a7f40-c265-4e42-a5c5-6f54e20b19df"
+      },
+      "period": {
+        "start": "2025-05-01T08:00:00.000+00:00",
+        "end": "2025-05-01T08:30:00.000+00:00"
+      }
+    },
+    "request": {
+      "method": "POST",
+      "url": "Encounter/55509231"
+    }
+  }, {
+    "fullUrl": "https://madie.cms.gov/Condition/55509232",
+    "resource": {
+      "resourceType": "Condition",
+      "id": "55509232",
```

Comment:
A purely numeric id will fail in HAPI, need to add a character to it (same for all test files)